api/pkg/apis/v1alpha1/model/instance.go

```diff
@@ -23,17 +23,16 @@ type (
 	// InstanceSpec defines the spec property of the InstanceState
 	// +kubebuilder:object:generate=true
 	InstanceSpec struct {
-		DisplayName  string            `json:"displayName,omitempty"`
-		Scope        string            `json:"scope,omitempty"`
-		Parameters   map[string]string `json:"parameters,omitempty"` //TODO: Do we still need this?
-		Metadata     map[string]string `json:"metadata,omitempty"`
-		Solution     string            `json:"solution"`
-		Target       TargetSelector    `json:"target,omitempty"`
-		Topologies   []TopologySpec    `json:"topologies,omitempty"`
-		Pipelines    []PipelineSpec    `json:"pipelines,omitempty"`
-		Generation   string            `json:"generation,omitempty"`
-		Version      string            `json:"version,omitempty"`
-		RootResource string            `json:"rootResource,omitempty"`
+		DisplayName string            `json:"displayName,omitempty"`
+		Scope       string            `json:"scope,omitempty"`
+		Parameters  map[string]string `json:"parameters,omitempty"` //TODO: Do we still need this?
+		Metadata    map[string]string `json:"metadata,omitempty"`
+		Solution    string            `json:"solution"`
+		Target      TargetSelector    `json:"target,omitempty"`
+		Topologies  []TopologySpec    `json:"topologies,omitempty"`
+		Pipelines   []PipelineSpec    `json:"pipelines,omitempty"`
+		Generation  string            `json:"generation,omitempty"`
```

Comment:
I thought generation was moved to metadata?

Reply:
The PR is not merged #303

Reply:
#303 was merged just now, please rebase.